src/supremm/summarize.py

```diff
@@ -1,54 +1,27 @@
-#!/usr/bin/env python3
-""" Summarize module """
+""" Definition of the summarize API """
+from abc import ABC, abstractmethod
 
-import datetime
-
-from ctypes import c_uint
-from pcp import pmapi
-import cpmapi as c_pmapi
-import time
-import logging
-import traceback
-from supremm.plugin import NodeMetadata
-from supremm.rangechange import RangeChange, DataCache
-from supremm.pcpcinterface import pcpcinterface
-
-import numpy
-import copy
-
-VERSION = "1.0.6"
+VERSION = "2.0.0"
```

Comment:
Why is this constant (1) changed, and defined in at least 3 different places?

Reply:
Resolved in:
https://github.com/ubccr/supremm/pull/259/commits/a34241f9050f4e6fcc5b9fb084199663ce3c9fe2
https://github.com/ubccr/supremm/pull/259/commits/e928967e850ddb22068e1e23ae33fa39e58e0b23
https://github.com/ubccr/supremm/pull/259/commits/c7b17840d6afddd9945104bdc48378a428543fb2